TESTS/mbedmicro-rtos-mbed/basic/main.cpp

```diff
@@ -1,38 +1,57 @@
 #include "mbed.h"
 #include "greentea-client/test_env.h"
 #include "rtos.h"
+#include "unity/unity.h"
 
 #if defined(MBED_RTOS_SINGLE_THREAD)
-  #error [NOT_SUPPORTED] test not supported
+#error [NOT_SUPPORTED] test not supported
 #endif
 
 #define TEST_STACK_SIZE 768
+#define ONE_MILLI_SEC 1000
 
-#define SIGNAL_PRINT_TICK 0x01
+volatile uint32_t callback_trigger_count = 0;
```

Comment:
I might be asking too much but I never had an idea what test does, can we add something here to the code file to explain?